get.sh

```diff
@@ -68,7 +68,18 @@ getPackage() {
     suffix=""
     case $uname in
     "Darwin")
-    suffix="-darwin"
+        arch=$(uname -m)
```

Comment:
I guess we'd want similar for [arkade](https://github.com/alexellis/arkade/releases)

Reply:
PR Opened for [arkade](https://github.com/alexellis/arkade/pull/600)